rfc11720

```diff
@@ -9,70 +9,113 @@ The objectives of the myTeams project RFC is to allow communication between myTe
+        -   team_uuid: UUID of the selected team
+        -   channel_uuid: UUID of the selected channel
+        -   thread_uuid: UUID of the selected thread
+
+- USE <SP> ["channel_uuid"] <SP> ["thread_uuid"] <EOL>
+-   This protocol is used to use specify a context thread
+        -   channel_uuid: UUID of the selected channel
+        -   thread_uuid: UUID of the selected thread
+
+- USE <SP> ["team_uuid"] <SP> ["channel_uuid"] <EOL>
+-   This protocol is used to use specify a context channel
+        -   team_uuid: UUID of the selected team
+        -   channel_uuid: UUID of the selected channel
+
+- USE <SP> ["team_uuid"] <EOL>
     -   This protocol is used to use specify a context team
         -   team_uuid: UUID of the selected team
-- USCH <SP> ["channel_uuid"] <EOL>
+
+- USE <SP> ["channel_uuid"] <EOL>
     -   This protocol is used to use specify a context channel
         -   channel_uuid: UUID of the selected channel
-- USTH <SP> ["thread_uuid"] <EOL>
+
+- USE <SP> ["thread_uuid"] <EOL>
     -   This protocol is used to use specify a context thread
         -   thread_uuid: UUID of the selected thread
-- CRTE <SP> ["team_name"] <SP> ["team_description"] <EOL>
+
+- CREA <SP> ["team_name"] <SP> ["team_description"] <EOL>
     -   This protocol is used to create a new team
         -   team_name: The name of the new team
         -   team_description: The description of the new team
-- CRCH <SP> ["channel_name"] <SP> ["channel_description"] <EOL>
+
+- CREA <SP> ["channel_name"] <SP> ["channel_description"] <EOL>
     -   This protocol is used to create a new channel
         -   channel_name: The name of the new channel
         -   channel_description: The description of the new channel
-- CRTH <SP> ["thread_name"] <SP> ["thread_description"] <EOL>
+
+- CREA <SP> ["thread_name"] <SP> ["thread_description"] <EOL>
     -   This protocol is used to create a new thread
         -   thread_name: The name of the new thread
         -   thread_description: The description of the new thread
-- CRRE <SP> ["comment_body"] <EOL>
+
+- CREA <SP> ["comment_body"] <EOL>
     -   This protocol is used to create a new reply
         -   comment_body: The comment to send
+
 - LSTE <EOL>
     -   This protocol is used to list all existing teams
-- LSCH <EOL>
+
+- LSTE <EOL>
     -   This protocol is used to list all existing channels
-- LSTH <EOL>
+
+- LSTE <EOL>
     -   This protocol is used to list all existing threads
-- LSRE <EOL>
+
+- LSTE <EOL>
     -   This protocol is used to list all existing replies
-- IFTE <EOL>
+
+- INFO <EOL>
     -   This protocol is used to display currently selected team infos
-- IFCH <EOL>
+
+- INFO <EOL>
     -   This protocol is used to display currently selected channel infos
-- IFTH <EOL>
+
+- INFO <EOL>
     -   This protocol is used to display currently selected thread infos
-- IFUS <EOL>
+
+- INFO <EOL>
     -   This protocol is used to display currently logged user infos
```

Comment:
It needs to be resumed. There is only one protocol but the uses will be different with the USE context